packages/build/bin/run-clean.js

```diff
@@ -42,7 +42,7 @@ function run(argv, options) {
         console.error('Skipping ' + f + ' as it is not inside the project');
       }
     } else {
-      if (!options.dryRun) fs.removeSync(f);
+      if (!options.dryRun) rimraf.sync(f);
```

Comment:
Building on top of my comment above, `f` should be renamed to something like `p`.

Reply:
To be fixed by https://github.com/strongloop/loopback-next/pull/1180